xgi/classes/function.py

```diff
@@ -903,3 +905,71 @@ def subfaces(edges, order=None):
 
         faces += faces_to_add
     return faces
+
+
+def complement(H):
+    """Returns the complement of hypergraph H.
+
+    The complement of an hypergraph will have the same nodes (same indexes) and
```

Comment:
it's "a" because the "h" is pronounced :) 
```suggestion
    The complement of a hypergraph will have the same nodes (same indexes) and
```